SokolTextGame/Commands.cs

```diff
@@ -27,6 +29,33 @@ public void Execute(string[] words, World world)
             Environment.Exit(0);
         }
     }
+    public class WhoAmI : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            if (words.Length == 1 && string.Join(" ", words) == "who") Console.Write("who, who? =)\n");
+            else if (words.Length == 3 && string.Join(" ", words) == "who am i") Console.Write( world?.player?.Description);
+            else Console.Write("I know the command \"who am i\"\n");
+        }
+    }
+    public class Look : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            if (words.Length == 1 && string.Join(" ", words) == "look") Console.Write("Look... What's next? You can look at your weapon\n");
```

Comment:
Also, could you add a check for "look at" (two words) after this line? In that case it could say `Look at... what?`. This is just to make the game a little bit more interactive.